packages/embeds/embed-core/index.html

```diff
@@ -90,6 +90,7 @@
     </div>
     <span style="display: block;" ><a href="?only=all">Render All embeds together</a><small> - It would be slow to load</small></span>
     <div>
+      <a href="?only=ns:floatingButton">Floating Popup</a>
```

Comment:
There was non dedicated link to reach to test out just the Floating popup(in zen mode)